frontend/src/common/FlashMessage.react.tsx

```diff
@@ -16,9 +28,26 @@ export default class FlashMessage extends React.Component<FlashMessageProps, {}>
 
     return (
       <div className={this.props.code}>
-        <p>{this.props.message}</p>
+        <p>
+          {this.props.message} {' '}
+          {this.props.details
+            ? <a
+                className={'FlashMessage-detailsLink' + (this.state.showDetails ? '-displayed' : '-hidden')}
+                href=""
```

Comment:
How is empty `href` attribute rendered ? Is it valid to leave empty but I think that placing `#` there will be more conventional.